services/ui-src/src/components/modals/DeleteEntityModal.tsx

```diff
@@ -57,7 +57,7 @@ export const DeleteEntityModal = ({
 
 interface Props {
   entityType: string;
-  selectedEntity: AnyObject;
+  selectedEntity: EntityShape | undefined;
```

Comment:
would this be better if it were optional rather than allowed to be undefined? Maybe it all comes out the same

Reply:
yup! this is a good catch, I missed it on the merge from Nick's changes.